include/alpaka/math/Complex.hpp

```diff
@@ -14,560 +14,569 @@
+        constexpr ALPAKA_FN_HOST_ACC auto pow(T const& x, Complex<U> const& y)
+        {
+            return pow(Complex<T>(x), y);
+        }
 
-    //! Projection onto the Riemann sphere
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename T>
-    constexpr ALPAKA_FN_HOST_ACC Complex<T> proj(Complex<T> const& x)
-    {
-        return std::proj(std::complex<T>(x));
-    }
+        //! Projection onto the Riemann sphere
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename T>
+        constexpr ALPAKA_FN_HOST_ACC Complex<T> proj(Complex<T> const& x)
+        {
+            return std::proj(std::complex<T>(x));
+        }
 
-    //! Sine
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename T>
-    constexpr ALPAKA_FN_HOST_ACC Complex<T> sin(Complex<T> const& x)
-    {
-        return std::sin(std::complex<T>(x));
-    }
+        //! Sine
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename T>
+        constexpr ALPAKA_FN_HOST_ACC Complex<T> sin(Complex<T> const& x)
+        {
+            return std::sin(std::complex<T>(x));
+        }
 
-    //! Hyperbolic sine
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename T>
-    constexpr ALPAKA_FN_HOST_ACC Complex<T> sinh(Complex<T> const& x)
-    {
-        return std::sinh(std::complex<T>(x));
-    }
+        //! Hyperbolic sine
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename T>
+        constexpr ALPAKA_FN_HOST_ACC Complex<T> sinh(Complex<T> const& x)
+        {
+            return std::sinh(std::complex<T>(x));
+        }
 
-    //! Square root
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename T>
-    constexpr ALPAKA_FN_HOST_ACC Complex<T> sqrt(Complex<T> const& x)
-    {
-        return std::sqrt(std::complex<T>(x));
-    }
+        //! Square root
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename T>
+        constexpr ALPAKA_FN_HOST_ACC Complex<T> sqrt(Complex<T> const& x)
+        {
+            return std::sqrt(std::complex<T>(x));
+        }
 
-    //! Tangent
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename T>
-    constexpr ALPAKA_FN_HOST_ACC Complex<T> tan(Complex<T> const& x)
-    {
-        return std::tan(std::complex<T>(x));
-    }
+        //! Tangent
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename T>
+        constexpr ALPAKA_FN_HOST_ACC Complex<T> tan(Complex<T> const& x)
+        {
+            return std::tan(std::complex<T>(x));
+        }
 
-    //! Hyperbolic tangent
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename T>
-    constexpr ALPAKA_FN_HOST_ACC Complex<T> tanh(Complex<T> const& x)
-    {
-        return std::tanh(std::complex<T>(x));
-    }
+        //! Hyperbolic tangent
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename T>
+        constexpr ALPAKA_FN_HOST_ACC Complex<T> tanh(Complex<T> const& x)
+        {
+            return std::tanh(std::complex<T>(x));
+        }
 
-    //! @}
+        //! @}
+    } // namespace internal
 
+    using internal::Complex;
```

Comment:
[Question, non-blocking] @AuroraPerego  is this the alias which makes the ADL possible?